mscore/menus.cpp

```diff
@@ -513,7 +513,7 @@ PalettePanel* MuseScore::newDynamicsPalettePanel(bool defaultPalettePanel)
             "pppppp", "ppppp", "pppp",
             "ppp", "pp", "p", "mp", "mf", "f", "ff", "fff",
             "ffff", "fffff", "ffffff",
-            "fp", "sf", "sfz", "sff", "sffz", "sfp", "sfpp",
+            "fp", "pf", "sf", "sfz", "sff", "sffz", "sfp", "sfpp",
```

Comment:
Could you please add "pf" to the `arrayDefault` below? I don't think that one should be normally visible to a user but it contains "fp" already so it would be consistent to add "pf" there too.

Reply:
Not being visible there is intentional, pepper and ffff are not visible either and this is even rarer

Reply:
And the more button shows it now, so no need to clutter thw normal palette

Reply:
I mean that `arrayDefault` is not normally visible to a user, I suggest adding it there just to be consistent about that.

Reply:
Isn't that the one the palette gets build from?

Reply:
No, `array1` is used for master palette, `arrayDefault` seems to be normally visible only to script tests.

Reply:
done